src/frontend/src/components/DynamicTooltip.tsx

```diff
@@ -0,0 +1,40 @@
+import { Tooltip, Typography } from '@mui/material';
```

Comment:
Might want to put "This file is part of NER's FinishLine and licensed under GNU AGPLv3." on top like some of the other components (copy the comment from them). Not sure if it's important or not but good to have